channels/alpha

```diff
@@ -86,20 +86,24 @@ spec:
     recommendedVersion: 1.11.10
     requiredVersion: 1.11.10
   kopsVersions:
+  - range: ">=1.19.0-alpha.1"
+    #recommendedVersion: "1.19.0-alpha.1"
+    #requiredVersion: 1.19.0
+    kubernetesVersion: 1.18.3
```

Comment:
Can you explain why this is 1.18.3? should we not k8s beta releases here? I see that kops 1.17 used 1.17.0-beta.2 here:

https://github.com/kubernetes/kops/commit/6559a0e0317d301a1aea6d8745fed694901326df#diff-37c59376fa8938637370b21d9540af69R72-R76

Reply:
Okay, changed.